src/Microsoft.Health.Fhir.Azure/ContainerRegistry/AzureContainerRegistryAccessTokenProvider.cs

```diff
@@ -33,6 +33,8 @@ public class AzureContainerRegistryAccessTokenProvider : IContainerRegistryToken
         private const string ExchangeAcrRefreshTokenUrl = "oauth2/exchange";
         private const string GetAcrAccessTokenUrl = "oauth2/token";
 
+        private static readonly Uri AcrTargetResourceUri = new Uri("https://containerregistry.azure.net/");
```

Comment:
It would make more sense to add a property in the ConvertDataConfiguration for this const value: "https://containerregistry.azure.net/" if it won't break anything.

Reply:
@ms-teli after speaking with Dustin, I am going to keep the value as the default "management.azure.com" value in order to avoid breaking any OSS scenarios. We will override this value in PaaS for the ACR-specific URI. I created a new field name in the config because "armResourceManagerId" does not make sense given the new URI value and its purpose (to acquire a token for ACR).